simulation/bcmn_simulation.py

```diff
@@ -54,9 +61,16 @@ def simulate(host_type):
                 if h.name in generated:
                     continue
                 host_amount = verifier_check_amount(h, verifier)
-                print(h.name + ' has ' + str(host_amount) + ' coin currently, target is: ' + str(a))
+                print(
+                    h.name +
+                    ' has ' +
+                    str(host_amount) +
+                    ' coin currently, target is: ' +
+                    str(a))
                 if (host_amount >= a):
-                    print(h.name + ' has enough coin, stopping generation for it')
+                    print(
+                        h.name +
+                        ' has enough coin, stopping generation for it')
```

Comment:
it's probably better to keep these as one line. At least don't break it immediatly after first openning parenthesis